mmv1/third_party/terraform/services/compute/data_source_google_compute_reservation.go

```diff
@@ -0,0 +1,63 @@
+package compute
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var (
+	computeReservationIdTemplate = "projects/%s/zones/%s/reservations/%s"
+	computeReservationLinkRegex  = regexp.MustCompile("projects/(.+)/zones/(.+)/reservations/(.+)$")
+)
```

Comment:
these look unused

```suggestion
```

Reply:
Went ahead and added `TestComputeReservationIdParsing` which does use these values in the latest commit pushed.